### DISCUSSION TITLE
无法预览office文件求解决方案

### DISCUSSION THREAD

**Discussion by wchangc** [AlistGo/alist#3832] [General]
File not found
The URL of the original file is not valid or the document is not publicly accessible.
Verify the URL is correct, then contact the document owner.

**Comment by qinguoluo** [comment]
同问

**Comment by sunnywangzi** [comment]
[看看文档](https://alist.nn.ci/zh/faq/why.html#为什么我不能预览-office-文件)

**Reply by sunnywangzi** [reply]
```
对于 Microsoft Office 在线查看器，您需要：

外部网络访问
文档访问地址不能直接使用 IP，需要通过域名访问，端口必须是 80/443 端口
文件的格式（必须是以下之一）：
Word：docx、docm、dotm、dotx
Excel：xlsx、xlsb、xls、xlsm
PowerPoint：pptx、ppsx、ppt、pps、pptm、potm、ppam、potx、ppsm
文档大小：Word 和 PowerPoint 文档必须小于 10 MB； Excel 必须小于 5 MB
```

**Reply by qinguoluo** [reply]
alist安装在通过路由器连接上网的个人电脑上，是不能使用Microsoft Office在线预览的吗？

**Reply by sunnywangzi** [reply]
如果外网不能访问的话，就是不行。建议参照文档自行搭建预览服务
